client/css/style.css

```diff
@@ -2302,7 +2302,7 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 
 	#viewport.menu-dragging #sidebar-overlay,
 	#viewport.menu-dragging #sidebar {
-		transition: none !important;
+		transition: box-shadow 160ms;
```

Comment:
It turns out `!important` was not necessary anymore, but `box-shadow` transition must be kept or it makes it blink.